src/pages/docs/concepts-invocations.mdx

```diff
@@ -0,0 +1,223 @@
+import { Callout } from "nextra/components"
+import { Card, Cards } from "nextra/components"
+
+# Introduction
+
+Golem is a durable computing platform that makes it simple to build and deploy
+highly reliable distributed systems.
+
+The fundamental building block of every Golem application is the
+[component](#concepts-components), which is instantiated to form 
+[workers](#concepts-workers). The only way to create or interact with workers is 
+through _invocations_.
+
+An _invocation_ is the act of invoking an exported function on a specific 
+worker, by supplying the function with its required input parameters. If the 
+invoked function has a return value, the invocation will return the value.
+
+Perhaps surprisingly for those new to Golem, every invocation occurs on a 
+_specific worker_, rather than more generally on a component. If an attempt is 
+made to invoke a function on a worker that does not exist yet, it will be 
+created through the act of invocation.
+
+Invocation is such a fundamental aspect of building useful applications on 
+Golem that in this section, we will discuss the origin of invocations, and 
+provide high-level instructions on how to perform invocations during 
+development, testing, and production scenarios.
+
+# Structure of Invocations
```

Comment:
I added a page (`function-names.mdx` in Reference) which describes very similar things - how to map WIT exports and resources into the function name syntax etc. We should decide if we want to keep that, or only use this page and then remove the other one and change links to point to here.

Reply:
I will link to your page since it's much more comprehensive and we can decide whether to unify them or not later.